src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,33 @@
+
+import React, { useEffect, useState } from 'react';
+import { View, Text, Button } from 'react-native';
+import { APIService } from '../services/APIService';
```

Comment:
There is no file - '../services/APIService' Please use src/api/LeadApi.ts file instead